pkg/api/response/response.go

```diff
@@ -253,12 +252,6 @@ func Error(status int, message string, err error) *NormalResponse {
 		data["message"] = message
 	}
 
-	if err != nil {
-		if setting.Env != setting.Prod {
-			data["error"] = err.Error()
```

Comment:
:meow_think: What kinds of information can those errors contain? I don't know the answer, but I assumed that we omit the error message from the production responses to avoid leaking sensitive or identifying data.

Reply:
Well since it is "if not production" -- this change would not change what is shown in prod would it?  ... :coffee:

Reply:
So, right now when in dev mode we "leak" the underlying error out into the error response (presumably to make it easier for devs to see what the error was vs needing to look at the logs), while in prod we do not (because it potentially includes sensitive information)

The proposed change won't have any impact on prod, but will mean that in dev mode we lose that output. I tend to agree that it's better to be consistent and that in dev we can (and should) look at the logs to find the error.

`data["error"]` won't already have something in it, because `data` is an empty map and we only populate `message`